CI/flatpak/com.obsproject.Studio.json

```diff
@@ -1,515 +1,105 @@
-      "buildsystem": "cmake-ninja",
-      "config-opts": [
-        "-DCMAKE_BUILD_TYPE=Release"
-      ],
-      "no-make-install": true,
-      "make-args": [
-        "libcef_dll_wrapper"
-      ],
-      "build-commands": [
-        "mkdir -p /app/cef/libcef_dll_wrapper",
-        "cp -R ./include /app/cef",
-        "cp -R ./Release /app/cef",
-        "cp -R ./Resources /app/cef",
-        "cp -R ./libcef_dll_wrapper/libcef_dll_wrapper.a /app/cef/libcef_dll_wrapper"
-      ],
-      "cleanup": [
-        "*"
-      ],
-      "sources": [
-        {
-          "type": "archive",
-          "url": "https://cdn-fastly.obsproject.com/downloads/cef_binary_5060_linux64.tar.bz2",
-          "sha256": "ac4e2a8ebf20700e4e36353e314f876623633dd5b474778a2548bb66bdbea11d"
-        }
-      ]
-    },
-    {
-      "name": "obs",
-      "buildsystem": "cmake-ninja",
-      "builddir": true,
-      "config-opts": [
-        "-DCMAKE_BUILD_TYPE=Release",
-        "-DENABLE_WAYLAND=ON",
-        "-DENABLE_BROWSER=ON",
-        "-DCEF_ROOT_DIR=/app/cef",
-        "-DUSE_XDG=ON",
-        "-DENABLE_ALSA=OFF",
-        "-DENABLE_PULSEAUDIO=ON",
-        "-DENABLE_JACK=ON",
-        "-DENABLE_RTMPS=ON",
-        "-DENABLE_VLC=OFF",
-        "-DENABLE_AJA=ON",
-        "-DENABLE_LIBFDK=ON"
-      ],
-      "secret-opts": [
-        "-DRESTREAM_CLIENTID=$RESTREAM_CLIENTID",
-        "-DRESTREAM_HASH=$RESTREAM_HASH",
-        "-DTWITCH_CLIENTID=$TWITCH_CLIENTID",
-        "-DTWITCH_HASH=$TWITCH_HASH",
-        "-DYOUTUBE_CLIENTID=$YOUTUBE_CLIENTID",
-        "-DYOUTUBE_CLIENTID_HASH=$YOUTUBE_CLIENTID_HASH",
-        "-DYOUTUBE_SECRET=$YOUTUBE_SECRET",
-        "-DYOUTUBE_SECRET_HASH=$YOUTUBE_SECRET_HASH"
-      ],
-      "post-install": [
-        "install -d /app/plugins",
-        "install -d /app/extensions/Plugins"
-      ],
-      "sources": [
-        {
-          "type": "dir",
-          "path": "../../"
-        }
-      ]
-    }
-  ]
+        "org.freedesktop.LinuxAudio.Plugins": {
+            "directory": "extensions/Plugins",
+            "version": "22.08",
+            "add-ld-path": "lib",
+            "merge-dirs": "vst",
+            "subdirectories": true,
+            "no-autodownload": true
+        }
+    },
+    "cleanup": [
+        "/share/man",
+        "*.la"
+    ],
+    "modules": [
+        "modules/10-mbedtls.json",
+        "modules/20-aom.json",
+        "modules/20-librist.json",
+        "modules/20-nv-codec.json",
+        "modules/20-srt.json",
+        "modules/20-svt-av1.json",
+        "modules/20-x264.json",
+        "modules/30-ffmpeg.json",
+        "modules/40-luajit.json",
+        "modules/50-jansson.json",
+        "modules/50-ntv2.json",
+        "modules/50-pipewire.json",
+        "modules/50-swig.json",
+        "modules/50-v4l-utils.json",
+        "modules/90-asio.json",
+        "modules/90-nlohmann-json.json",
+        "modules/90-websocketpp.json",
+        "modules/99-cef.json",
+        {
```

Comment:
Just a hypothetical question, in order to avoid mixing types in this array, could the final module ("obs") be split out into a separate file as well?

Reply:
I would prefer to avoid that, it's uncommon to have dependencies as separate modules but in our case it improves maintaince imo. But not the main app that must be the last module.